src/gui/tracks/TrackView.cpp

```diff
@@ -48,6 +49,9 @@
 
 namespace lmms::gui
 {
+/*! The width of the resize grip in pixels
+	*/
+const int RESIZE_GRIP_WIDTH = 6;
```

Comment:
Why not store this in `TrackView` as a `TrackView::ResizeGripWidth` instead? Also, the `UPPER_SNAKE_CASE` style should be reserved for macros only.